src/components/Games/story-game/level-selector.jsx

```
@@ -0,0 +1,32 @@
+import React from "react";
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)